packages/ckeditor5-ui/src/editorui/poweredby.ts

```diff
@@ -261,21 +274,52 @@ function getLowerCornerPosition( getBalloonLeft: ( rootRect: Rect, balloonRect:
 		}
 
 		const isRootNarrow = rootRect.width < NARROW_ROOT_WIDTH_THRESHOLD;
-		const balloonTop = rootRect.bottom - balloonRect.height - CORNER_OFFSET;
+
+		let balloonTop;
+
+		/* istanbul ignore else -- @preserve */
+		if ( config.position === 'inside' ) {
+			balloonTop = rootRect.bottom - balloonRect.height;
+		}
+		else {
+			balloonTop = rootRect.bottom - balloonRect.height / 2;
+		}
+
+		balloonTop -= config.verticalOffset;
+
 		const balloonLeft = getBalloonLeft( rootRect, balloonRect );
-		const newBalloonRect = balloonRect.clone().moveTo( balloonLeft, balloonTop );
 
-		// The watermark cannot be positioned in this corner because the corner is not quite visible.
-		/* istanbul ignore next -- @preserve */
-		if ( newBalloonRect.getIntersectionArea( visibleRootRect ) < newBalloonRect.getArea() ) {
-			return OFF_THE_SCREEN_POSITION;
+		/* istanbul ignore else -- @preserve */
+		if ( config.position === 'inside' ) {
```

Comment:
Logic for `getLowerCornerPosition()` function is getting crowdy. I'll make a followup to refactor it a bit.